### PR TITLE
Updating Chapter 8 with more inclusive terms

### PR DESCRIPTION
**Pull request by rmhyman**
The terms master/slave should not be propagated for future technologists to absorb. Made changes to text and images (some resizing may be necessary). I'm not sure this terminology is being used in subsequent chapters, but I'm more than happy to update them as I encounter them.

### CODE REVIEW

File: microbit/src/08-i2c/README.md [hunk 1]
```diff
@@ -18,15 +18,15 @@ uses two lines to exchange data: a data line (SDA) and a clock line (SCL). Becau
...
 <p align="center">
-<img class="white_bg" height=180 title="I2C bus" src="https://upload.wikimedia.org/wikipedia/commons/3/3e/I2C.svg">
+<img class="white_bg" height=270 title="I2C bus" src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/04/I2C_controller-target.svg/440px-I2C_controller-target.svg.png">
 </p>
 
-This protocol uses a *master* *slave* model where the master is the device that *starts* and
-drives the communication with a slave device. Several devices, both masters and slaves, can be
-connected to the same bus at the same time. A master device can communicate with a specific slave
+This protocol uses a *controller* *target* model where the controller is the device that *starts* and
+drives the communication with a target device. Several devices, both conntrollers and targets, can be
```
Comment (adamgreig) [inline microbit/src/08-i2c/README.md:25]:
```suggestion
drives the communication with a target device. Several devices, both controllers and targets, can be
```

**Review by adamgreig — CHANGES_REQUESTED** [review]
Thanks! These are also the official terms for I2C now; the spec was updated last year.

I noticed one typo, and also I think it would be better if we could use the SVG version of the new images from Wikipedia - could you try changing to those please?

**Comment by rmhyman** [comment]
> Thanks! These are also the official terms for I2C now; the spec was updated last year.
> 
> I noticed one typo, and also I think it would be better if we could use the SVG version of the new images from Wikipedia - could you try changing to those please?

That's good to know that the spec was updated 😃 . I've fixed the typo and updated the images to use the SVG file instead of the PNG version.

**Review by adamgreig — APPROVED** [review]
Thanks!

bors r+

**Pull request merged**